pages/common/mongoimport.md

```diff
@@ -0,0 +1,24 @@
+# mongoimport
+
+> Imports content from a JSON, CSV, or TSV file into a MongoDB database.
+> More information: <https://docs.mongodb.com/database-tools/mongoimport/>.
+
+- Import a JSON file into a specific collection:
+
+`mongoimport --file={{path/to/file}} --uri={{mongodb_uri}} --collection={{collection_name}}`
+
+- Import a CSV using the first line of the file to determine property names:
```

Comment:
What about?
```suggestion
- Import a CSV file using the first line of the file to determine property names:
```
It would be similar to the `JSON file` expression above.

Reply:
`as field names` also seems better to me, see [the MongoDB doc about documents](https://docs.mongodb.com/manual/core/document/).

Reply:
Ah, I see.  I wasn't aware that there was a specific name.  Let's use "field" then. 👍🏻